### DISCUSSION TITLE
[Feature Request]: Cluster autoscaler labels/taints per nodepool

### DISCUSSION THREAD

**Discussion by Silvest89** [mysticaltech/terraform-hcloud-kube-hetzner#1040] [Ideas]
### Description

Would be nice to be able to set different taints/labels per autoscaler nodepool

**Comment by ricristian** [comment]
You already have this option in kube.tf.example or I am missing something ? 

https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/blob/master/kube.tf.example#L269C2-L280C1

**Comment by Silvest89** [comment]
@ricristian Yes but if you set it there it goes for all autoscaler node pools.

Something like this would be nice
```
  autoscaler_nodepools = [
    {
      name        = "autoscaled-small"
      server_type = "cpx11"
      location    = "fsn1"
      min_nodes   = 0
      max_nodes   = 1
      labels = []
      taints = []
    }
  ]
```

**Comment by Silvest89** [comment]
Implemented in https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/pull/1012
Waiting for upstream [autoscaler](https://github.com/kubernetes/autoscaler)
